cyclops/report/examples/cyclops_mock_template.ipynb

```diff
@@ -21,126 +21,13 @@
     "from sklearn.model_selection import train_test_split\n",
     "from sklearn.pipeline import Pipeline\n",
     "\n",
-    "from cyclops.report.report import ModelCardReport"
+    "from cyclops.report import ModelCardReport"
```

Comment:
@fcogidi Would it be possible to update the mock template to use the kaggle dataset example instead of the credit card fraud dataset? https://vectorinstitute.github.io/cyclops/api/tutorials_use_cases.html#in-hospital-mortality-prediction. So basically you can move this jupyter notebook to docs/source/tutorials/mimiciii, and then have the notebook run every time to generate a sample report. I can then simply add the sample report to be generated as an artifact from the docs build job, and published to the webpage under tutorials/documentation.

Reply:
In a separate notebook or the same one as the mimiciii tutorial? We'd need to evaluate a model anyway.

Reply:
Same notebook is also fine.

Reply:
I added the report API to the mimic-iii tutorial, but I don't think the API is stable enough for a tutorial.